[2024-07-01T19:20:00.231+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-01T19:20:00.285+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: purchase_dag.table_creation manual__2024-07-01T19:19:56.354665+00:00 [queued]>
[2024-07-01T19:20:00.302+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: purchase_dag.table_creation manual__2024-07-01T19:19:56.354665+00:00 [queued]>
[2024-07-01T19:20:00.303+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-01T19:20:00.328+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): table_creation> on 2024-07-01 19:19:56.354665+00:00
[2024-07-01T19:20:00.339+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=187) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-01T19:20:00.338+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'purchase_dag', 'table_creation', 'manual__2024-07-01T19:19:56.354665+00:00', '--job-id', '524', '--raw', '--subdir', 'DAGS_FOLDER/purchase_dag.py', '--cfg-path', '/tmp/tmptoogetsm']
[2024-07-01T19:20:00.340+0000] {standard_task_runner.py:63} INFO - Started process 188 to run task
[2024-07-01T19:20:00.341+0000] {standard_task_runner.py:91} INFO - Job 524: Subtask table_creation
[2024-07-01T19:20:00.356+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-01T19:20:00.404+0000] {task_command.py:426} INFO - Running <TaskInstance: purchase_dag.table_creation manual__2024-07-01T19:19:56.354665+00:00 [running]> on host 0a37133e4d82
[2024-07-01T19:20:00.557+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='purchase_dag' AIRFLOW_CTX_TASK_ID='table_creation' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T19:19:56.354665+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-01T19:19:56.354665+00:00'
[2024-07-01T19:20:00.558+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-01T19:20:00.586+0000] {logging_mixin.py:188} INFO - Couldn't execute Bronze table creation due to exception: relation "bronze_purchase_load" already exists
[2024-07-01T19:20:00.587+0000] {logging_mixin.py:188} INFO - Couldn't execute Silver table creation due to exception: relation "bronze_purchase_load" already exists
[2024-07-01T19:20:00.588+0000] {logging_mixin.py:188} INFO - Couldn't execute Gold table creation due to exception: relation "bronze_purchase_load" already exists
[2024-07-01T19:20:00.588+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-01T19:20:00.592+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-01T19:20:00.606+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=purchase_dag, task_id=table_creation, run_id=manual__2024-07-01T19:19:56.354665+00:00, execution_date=20240701T191956, start_date=20240701T192000, end_date=20240701T192000
[2024-07-01T19:20:00.639+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-01T19:20:00.689+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-01T19:20:00.694+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
